src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+interface ButtonProps {
+  children: React.ReactNode;
+}
+
+const styles = {
+  snazzyButton: {
+
+    backgroundColor: 'blue',
+    color: 'white',
+    padding: '10px 20px',
+    borderRadius: '5px',
+
+  },
+
+};
+
+const useStyles = createUseStyles(styles);
+
+interface SnazzyButtonProps {
+  isSnazzy: boolean;
+}
+
+const SnazzyButton: React.FC<SnazzyButtonProps> = ({ children, ...rest }) => {
```

Comment:
where is the import for isSnazzy?

Reply:
You imported this component, and called it isSnazzy... 
You originally had a 'prop' where children was.. Like this: 
```suggestion
const SnazzyButton: React.FC<SnazzyButtonProps> = ({ isSnazzy, children, ...rest }) => {
```